buildSrc/build.gradle.kts

```diff
@@ -16,7 +16,7 @@ dependencies {
   implementation("com.google.guava:guava:31.1-jre")
   implementation("com.google.code.gson:gson:2.10.1")
 
-  implementation(platform("org.http4k:http4k-bom:4.39.0.0"))
+  implementation(platform("org.http4k:http4k-bom:4.36.0.0"))
```

Comment:
This causes compilating error in my case

Reply:
is your kotlin version 1.8 in AS? it should really use the latest version as our kotlin version is 1.8